### PR TITLE
Documentation: Fix field_name for "product_title"

### PR DESCRIPTION
**Pull request by muffl0n**

### CODE REVIEW

File: docs/topics/loaders.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ chapter <topics-items>`::
     def parse(self, response):
         l = ItemLoader(item=Product(), response=response)
         l.add_xpath('name', '//div[@class="product_name"]')
-        l.add_xpath('name', '//div[@class="product_title"]')
+        l.add_xpath('title', '//div[@class="product_title"]')
```
[the remark this answers is not in the document]
Comment (muffl0n) [reply docs/topics/loaders.rst]: But it's still bogus to assign the xpath for "product_title" to a field named "name". Especially if it is assigned twice.

[the remark this answers is not in the document]
Comment (kmike) [reply docs/topics/loaders.rst]: It is not assigned twice, see explanation below this code snippet.

[the remark this answers is not in the document]
Comment (muffl0n) [reply docs/topics/loaders.rst]: Oh my, sorry for that. I did completely miss that part. :/

[the remark this answers is not in the document]
Comment (kmike) [reply docs/topics/loaders.rst]: No worries, it is not always obvious how ItemLoaders work, unless you have an experience with them. Is it OK to close this pull request?

[the remark this answers is not in the document]
Comment (muffl0n) [reply docs/topics/loaders.rst]: Absolutely! Keep up the good work! :)

**Pull request closed without merging**